modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -262,9 +260,21 @@ definition {
 
 			CommerceEntry.clickEntryDropdownMenuItem(
 				entryName = "Value1",
-				menuItem = "Toggle Default");
+				menuItem = "Edit");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Default",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
 
 			Alert.viewSuccessMessage();
+
+			SelectFrameTop();
+
+			Click(locator1 = "Icon#CLOSE");
```

Comment:
After saving, please add an assert to verify if the toggle has successfully changed the status.